core/src/listener/listener.ts

```diff
@@ -242,7 +242,7 @@ function latestJob({
         }
         logger.info(logPrefix)
       } else {
-        logger.info(`${logPrefix} noop`)
+        logger.debug(`${logPrefix} noop`)
```

Comment:
We have to convert two more logs

* https://github.com/Bisonai/orakl/blob/7726b62013ad42ff72299134f66ea6319f4a3454/core/src/listener/listener.ts#L243
* https://github.com/Bisonai/orakl/blob/7726b62013ad42ff72299134f66ea6319f4a3454/core/src/listener/listener.ts#L304

Reply:
thought you were indicating only this specific log, I'll update it too